cron/vidyo2MozDef.py

```diff
@@ -42,18 +42,22 @@ class MozDefEvent():
     log['severity'] = 'INFO'
     log['summary'] = None
     log['category'] = 'event'
+    log['source'] = 'vidyo'
     log['tags'] = list()
     log['details'] = dict()
 
-    def __init__(self, url='http://localhost/events', summary=None, category='event', severity='INFO', tags=[], details={}):
+
+    def __init__(self, url='http://localhost/events', summary=None, category='event', severity='INFO', source='vidyo', tags=[], details={}):
```

Comment:
Added source here, because I was not sure if "type" could be used when stating self.type = type in that fashion.
I felt maybe we could go  back and change it to add casting this value into a type field, but for now added it as source. pinging pwnbus for insight.